### PR TITLE
dont crash observing run page for targets that dont rise

### PR DESCRIPTION
**Pull request by dannygoldstein**
Fixes https://github.com/fritz-marshal/fritz-beta-feedback/issues/44

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]
LGMT, but I'd run it by @stefanv.

**Review by stefanv — APPROVED** [review]

File: skyportal/handlers/api/observingrun.py [hunk 1]
```diff
@@ -156,8 +163,8 @@ def get(self, run_id=None):
                 set_times = run.set_time(targets).isot
 
                 for d, rt, st in zip(data["assignments"], rise_times, set_times):
-                    d["rise_time_utc"] = rt
-                    d["set_time_utc"] = st
+                    d["rise_time_utc"] = rt if rt is not np.ma.masked else ''
```
Comment (stefanv) [inline skyportal/handlers/api/observingrun.py:166]: Can we adapt one of our existing test cases to have such data?

**Comment by dannygoldstein** [comment]
working on a test now

**Comment by dmitryduev** [comment]
@dannygoldstein is this ready for merging?

**Comment by dannygoldstein** [comment]
@dmitryduev  Yep.

**Pull request merged**